CHANGELOG

```diff
@@ -1,2 +1,77 @@
-1.1.2
-- Changed to nested module style see: https://github.com/bitcrowd/rubocop-bitcrowd/pull/4
+# Rubocop Bitcrowd Changelog
+
+Presented in reverse chronological order.
+
+## master
+
+https://github.com/bitcrowd/rubocop-bitcrowd/compare/v1.2.1...HEAD
+
+* Put high-level summary here
+
+### Deprecations:
+
+* Put deprecations here
+
+### Potentially breaking changes:
+
+* Put potentially breaking changes here
+
+### New features:
+
+* added `staging` to the list of known environments
+* enforce frozen_string_literal comment
+* Put new features here
```

Comment:
See my previous comment for making this easier distinguishable from bullet points.